fil-proofs-tooling/src/bin/benchy/stacked.rs

```diff
@@ -313,9 +317,39 @@ where
     report.outputs.total_report_wall_time_ms = total_wall_time.as_millis() as u64;
     report.outputs.total_report_cpu_time_ms = total_cpu_time.as_millis() as u64;
 
+    augment_with_op_measurements(&mut report);
+
     Ok(report)
 }
 
+#[cfg(not(feature = "metrics"))]
+fn augment_with_op_measurements(mut _report: &mut Report) {}
+
+#[cfg(feature = "metrics")]
+fn augment_with_op_measurements(mut report: &mut Report) {
+    // drop the tx side of the channel, causing the iterator to yield None
+    // see also: https://doc.rust-lang.org/src/std/sync/mpsc/mod.rs.html#368
+    *OP_MEASUREMENTS.0.lock().expect("failed to acquire mutex") = None;
```

Comment:
you can use `.take` to do this in a nicer way

Reply:
good call. updated.